docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -0,0 +1,550 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8d3e3a9c",
+   "metadata": {},
+   "source": [
+    "# Create and Use Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bfe1aea0",
+   "metadata": {},
+   "source": [
+    "Functions are the basic building blocks of MLRun - they are essentially a Python object that knows how to run locally or on a Kubernetes cluster. This section will cover how to create and customize an MLRun Function as well as common parameters across all functions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d68f026",
+   "metadata": {},
+   "source": [
+    "**In this section:**\n",
+    "- [**Functions Overview**](#functions-overview)\n",
+    "- [**Functions and Projects**](#functions-and-projects)\n",
+    "- [**Creating Functions**](#creating-functions)\n",
+    "- [**Customizing Functions**](#customizing-functions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "392e01ae",
+   "metadata": {},
+   "source": [
+    "<a id=\"functions-overview\"></a>\n",
+    "## Functions Overview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98df9683",
+   "metadata": {},
+   "source": [
+    "MLRun Functions are used to run jobs, deploy models, create pipelines, and more. There are various kinds of MLRun functions with different capabilities, however, there are commonalities across all functions. In general, an MLRun function looks like the following:\n",
+    "\n",
+    "![MLRun Function](../_static/images/mlrun_function_diagram.png)\n",
+    "\n",
+    "You can read more about MLRun Functions [**here**](./functions.html). Each parameter and capability will be explained in more detail in the following sections [**Creating Functions**](#creating-functions) and [**Customizing Functions**](#customizing-functions)."
```

Comment:
"You can read more about MLRun functions [**here**](./functions.html). Each parameter and capability is explained in more detail in the following sections [**Creating functions**](#creating-functions) and [**Customizing functions**](#customizing-functions)."